src/graphql/query/thanksPage.graphql

```diff
@@ -132,11 +132,11 @@ query checkoutReceipt($checkoutId: Int!, $visitorId: String) {
 			key
 			value
 		}
-		ftd_message_enable: uiConfigSetting(key: "thanks_page_ftd_message_enable") {
+		ftd_message_enable: uiConfigSetting(key: "ftd_message_enable") {
 			key
 			value
 		}
-		ftd_amount: uiConfigSetting(key: "thanks_page_ftd_amount") {
+		ftd_amount: uiConfigSetting(key: "ftd_message_amount") {
```

Comment:
Why were these two keys changed?

Reply:
Because we are now using the same settings for both pages, checkout and thanks page.